client/src/components/Category.jsx

```diff
@@ -18,7 +18,8 @@ class Category extends React.Component {
     }
 
     getEventsByCategory(categoryId) {
-        return axios.get(`/api/categories/${categoryId}`)
+        // TODO: Replace hardcoded localhost IP with environmental variable
+        return axios.get(`http://localhost:3016/api/categories/${categoryId}`)
             .then((response) => {
```

Comment:
So, I see you started an Instance in AWS.  This path will eventually need to change again.  The Path will be the AWS url.

I'll go over this with you tomorrow.